lib/web/files.go

```diff
@@ -106,7 +106,7 @@ func (f *fileTransfer) download(req fileTransferRequest, httpReq *http.Request,
 		return trace.Wrap(err)
 	}
 
-	err = tc.ExecuteSCP(httpReq.Context(), cmd)
+	err = tc.ExecuteSCP(httpReq.Context(), req.server, cmd)
```

Comment:
if not too much trouble, can we rename `server` to `serverID`?

Reply:
No trouble at all! Fixed. much more clear this way.